blog/2025-06-15-temporal-impl-1.md

```diff
@@ -0,0 +1,439 @@
+mention `JsObject`, like `JsValue`, is implementation defined as well?
+
+This is where `temporal_rs`'s Partial API comes in.
+
+It turns out that while property bags in JavaScript can have various
+fields set, there is still a general shape for the fields that can be
+provided and validated in Temporal.
+
+So in order to support this in `temporal_rs`, a "partial" component
+exists for each of the components that can then be provided to that
+components `from_partial` method.
+
+With this, we have fully implemented support for the `from` method in
+`temporal_rs`:
+
+```rust
+use core::str::FromStr;
+use temporal_rs::{PlainDate, PlainDateTime, partial::PartialDate};
+let pdt = PlainDateTime::try_new_iso(2025, 1, 1)?;
+// We can use the `PlainDateTime` (`ZonedDateTime` / `PlainDate` are also options).
+let pd_from_pdt = PlainDate::from(pdt);
+// We can use a `str`.
+let pd_from_string = PlainDate::from_str("2025-01-01")?;
+// We can use a `PartialDate`.
+let pd_from_partial = PlainDate::from_partial(PartialDate {
+    year: Some(2025),
+    month: Some(1),
+    day: Some(1),
+    ..Default::default()
+});
+```
+
+**NOTE:** there may be updates to `PartialDate` in the future (see
+[boa-dev/temporal #349](https://github.com/boa-dev/temporal/issues/349)
+for more information).
+
+## Elephant in the room: time zones
+
+So far we have not discussed time zones, and -- surprise! -- we aren't
+going to ... yet. It's not because they aren't super cool and
+interesting and everyone totally 100% loves them. No, time zones aren't
+in this post, because they are still being polished and deserve an
+entire post of their own.
+
+So stay tuned for our next post on implementing Temporal! The one where
+we'll hopefully go over everyone's favorite subject, time zones; and
+answer the question that some of you may have if you happen to take a
+glance at `temporal_rs`'s docs or try out our `no_std` support: what in
+the world is a provider API?
+
+## Conclusion
+
+In conclusion, we're implementing Temporal in Rust to support engine
+implementors as well as to have the API available in native Rust in
+general.
+
+Boa currently sits at a [90% conformance rate][boa-test262] for Temporal
+completely backed by `temporal_rs` v0.0.8, and we're aiming to be 100%
+conformant before the end of the year.
+
+If you're interested in trying Temporal using Boa, you can use it in
+Boa's CLI or enable it in `boa_engine` with the `experimental` flag.
+
+Outside of Boa's implementation, `temporal_rs` has implemented or
+supports the implementation for a large portion of the Temporal's API in
+native Rust.
+
+If you're interested in trying out `temporal_rs`, feel free to add to
+your project with Cargo:
+
+```bash
+cargo add temporal_rs
+```
+
+or by adding the below to your `Cargo.toml`
+
+```toml
+temporal_rs = "0.0.9"
+```
+
+A FFI version of temporal is also available for C and C++ via
+`temporal_capi`.
+
+## General note on API stability
```

Comment:
Could be a footnote instead? Seems weird to have an additional section after the "conclusion"

Reply:
Tried making it a markdown footnote. It didn't seem to render on docusaurus.